app/views/results.nunjucks

```diff
@@ -9,34 +9,84 @@
 {% block content %}
 <a href="{{ SITE_ROOT }}/find-help?context={{ context }}" class="link-back">Back to find a pharmacy</a>
 
-<div class="results u-border u-nomargin">
+<div class="grid-row">
+
+  <h1 class="sr-only">Search results</h1>
+
   {% if openServices.length === 0 and nearbyServices.length === 0 %}
-  <p class="nout-at-all">NOTHING TO SEE HERE</p>
+
+  <div class="results">
+    <div class="results-block">
+      <h2 class="results-none">There are no pharmacies within 20 miles of your location</h2>
+      <p>Call 111 for:</p>
+      <ul>
+        <li>medical advice if you aren’t in an emergency</li>
+        <li>advice on whether you need urgent care</li>
+        <li>more information about a specific health condition</li>
+      </ul>
+      <p>If you need emergency help, call 999.</p>
+    </div>
+  </div>
+
   {% else %}
```

Comment:
This can go and the line 33 can be changed to `{% elif openServices.length > 0 %}` and line 87 can be removed - I think.